[2022-09-03T13:00:25.637+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: weather_data.insert_weather_mkp scheduled__2022-09-03T12:00:00+00:00 [queued]>
[2022-09-03T13:00:25.648+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: weather_data.insert_weather_mkp scheduled__2022-09-03T12:00:00+00:00 [queued]>
[2022-09-03T13:00:25.649+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-03T13:00:25.650+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-03T13:00:25.651+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-03T13:00:25.668+0000] {taskinstance.py:1389} INFO - Executing <Task(PostgresOperator): insert_weather_mkp> on 2022-09-03 12:00:00+00:00
[2022-09-03T13:00:25.675+0000] {standard_task_runner.py:52} INFO - Started process 125 to run task
[2022-09-03T13:00:25.678+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'insert_weather_mkp', 'scheduled__2022-09-03T12:00:00+00:00', '--job-id', '269', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpyrf7csst', '--error-file', '/tmp/tmpxe16ffqz']
[2022-09-03T13:00:25.679+0000] {standard_task_runner.py:80} INFO - Job 269: Subtask insert_weather_mkp
[2022-09-03T13:00:25.726+0000] {task_command.py:371} INFO - Running <TaskInstance: weather_data.insert_weather_mkp scheduled__2022-09-03T12:00:00+00:00 [running]> on host f432245fdbd9
[2022-09-03T13:00:25.820+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_data
AIRFLOW_CTX_TASK_ID=insert_weather_mkp
AIRFLOW_CTX_EXECUTION_DATE=2022-09-03T12:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-03T12:00:00+00:00
[2022-09-03T13:00:25.829+0000] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-09-03T13:00:25.834+0000] {sql.py:315} INFO - Running statement: insert into weather(
                name,
                temp,
                temp_feels_like,
                temp_min,
                temp_max,
                description,
                wind_speed,
                humidity,
                pressure,
                sunset,
                sunrise
            ) values (
                'Майкоп',
                35.41,
                34.64,
                35.41,
                35.41,
                'ясно',
                1.63,
                27,
                1009,
                1662220458,
                1662173052
            ), parameters: None
[2022-09-03T13:00:25.836+0000] {sql.py:324} INFO - Rows affected: 1
[2022-09-03T13:00:25.846+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=insert_weather_mkp, execution_date=20220903T120000, start_date=20220903T130025, end_date=20220903T130025
[2022-09-03T13:00:25.892+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-03T13:00:25.913+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
